### PR TITLE
email search is now case insensitive

### PR DESCRIPTION
**Pull request by Espe-on**
enforce .toLower on searches to make case insenstitive

### CODE REVIEW

**Review by MichaelWalker — CHANGES_REQUESTED** [review]
This is dangerous.

We've got `Sasha.Burgoyne@softwire.com` in the database already.
What happens if someone else comes along and adds `sasha.burgoyne@softwire.com`?

**Comment by Espe-on** [comment]
yeah I should redo this in the DB shouldn't I

File: TestSwitchApi/TestSwitchApi/Repositories/AdminRepo.cs [hunk 1]
```diff
@@ -2,21 +2,36 @@
...
+            var salt = _passwordService.GenerateSalt();
+            var newAdminUser = new AdminUserDataModel()
+            {
+                Email = email,
+                PasswordSalt = salt,
+                HashedPassword = _passwordService.HashPassword(password, salt),
+            };
+            _context.AdminUsers.Add(newAdminUser);
+            _context.SaveChanges();
+        }
```
Comment (MichaelWalker) [inline TestSwitchApi/TestSwitchApi/Repositories/AdminRepo.cs:30]: yeah - much nicer to have it here - would make it easy to add a 'CreateAdmin' endpoint later

File: TestSwitchApi/TestSwitchApi/Models/DataModels/AdminUserDataModel.cs [hunk 2]
```diff
@@ -8,7 +8,13 @@ public class AdminUserDataModel
         [Key]
         [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
         public int Id { get; set; }
-        public string Email { get; set; }
+
+        public string Email
+        {
+            get => Email;
+            set => Email.ToLower();
```
Comment (MichaelWalker) [inline TestSwitchApi/TestSwitchApi/Models/DataModels/AdminUserDataModel.cs]: Does this work? I was expecting you to need to use `set => value.ToLower`.

**Review by MichaelWalker — APPROVED** [review]
Fine with this if it works... but those properties look a little suspicious to me! 😂

File: TestSwitchApi/TestSwitchApi/Models/DataModels/AdminUserDataModel.cs (hunk 2 above)
Comment (Espe-on) [reply TestSwitchApi/TestSwitchApi/Models/DataModels/AdminUserDataModel.cs]: somehow an old commit has sneaked in here

**Pull request merged**